tests/pools/test_pool_puzzles_lifecycle.py

```diff
@@ -242,7 +242,7 @@ def test_pool_lifecycle(self):
         # filter for only the singleton solution
         singleton_coinsol: CoinSpend = list(
             filter(
-                lambda e: e.coin.amount == START_AMOUNT,
+                lambda e: e.coin.amount != START_AMOUNT,
```

Comment:
this is perhaps a bit unintuitive. the idea here is to pick the larger coin, to avoid `Err.MINTING_COIN` further down.

Reply:
This would make a great comment

Reply:
I'm not sure I understand this. What's failure case causes `Err.MINTING_COIN`?

Reply:
I see, it refers to an imbalance of outgoing vs incoming that causes unapproved coins to be minted.

Reply:
exactly. the `START_AMOUNT` is 1023, but the dummy coin we try to create further down has amount 1337, which isn't allowed (and it used to not be checked this early)